packages/admin-ui/ui/src/domain/orders/details/allocations/edit-allocation-modal.tsx

```diff
@@ -72,11 +72,15 @@ const EditAllocationDrawer = ({
   const handleDelete = () => {
     deleteReservation(undefined, {
       onSuccess: () => {
-        notification("Success", "Allocation deleted successfully", "success")
+        notification(
+          "Allocation was deleted",
+          "The allocated items have been released.",
+          "success"
+        )
         close()
       },
       onError: () => {
-        notification("Errors", "Failed to deleted ", "success")
+        notification("Errors", "Failed to delete the allocation ", "error")
```

Comment:
very-nit(non-blocking): Should the title not be "Error" instead of "Errors", or even better a more descriptive title, like what you have done for the onSuccess.

Reply:
Fixed "Errors" to "Error", but as far as I can tell, the "Error" title is very much standard in our codebase so far, I don't know if I'd want to have just the one be more verbose.

Reply:
Let's revisit our error messages some other time and merge this now thanks! 👍